conduct/COC_POLICY.md

```diff
@@ -0,0 +1,109 @@
+This document defines the OpenJS Foundation's code of conduct policy.
+
+If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
+
+***
+
+# Code of Conduct Policy
+
+The OpenJS Foundation (OpenJSF) delegates to the Cross Project Council (CPC), via its [charter][], the responsibility of choosing, maintaining, and enforcing a Code of Conduct (CoC) for all of its projects, collab spaces, and for any related activities.
+
+## Code of Conduct
+
+The OpenJSF's code of conduct is located at [code-of-conduct.openjsf.org][CoC]. It is based on the [Contributor Covenant](https://www.contributor-covenant.org/).
+
+A common code of conduct allows a common escalation and appeal process and allows better support of projects in their efforts to apply the code of conduct.
+
+All OpenJSF projects, collab spaces, and any activities related to the OpenJSF are operating under this code of conduct unless otherwise stated. (For example, events organized in partnership with other organizations might be operating under a different code of conduct.)
+
+Projects joining the foundation that already have a code of conduct will need to adopt the OpenJSF's CoC within three (3) months. Projects that don't already have a code of conduct operate under the OpenJSF's CoC as soon as they enter incubation.
```

Comment:
Should this be mentioned here? This should be said on this document (https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md) instead of here. This sort of information feels only relevant for projects joining the Foundation.

Reply:
I understand that perspective and agree that this information needs to be repeated in the onboarding-related doc, but the decision to give some leniency to projects joining the foundation is a policy decision and belongs here, imho.

Reply:
Yes! Would you want to take a stab at it?

Reply:
wdym? do you want me to make the proposed changes? 👀

Reply:
Yeah, I was thinking mainly about updating the issue template [here](https://github.com/openjs-foundation/project-status/blob/main/.github/ISSUE_TEMPLATE/project-onboarding-checklist-template.md) alongside updating `PROJECT_PROGRESSION.md`.

Reply:
I can work on that, can we add a separate "tasklist" for this on the main CoC refactor issue? (If you don't mind doing that, I can then start working on a PR)

Reply:
_also feel free to resolve this specific thread once you've added the task on the issue and assigned it to me 🙇_

Reply:
See https://github.com/openjs-foundation/cross-project-council/issues/1255.